config/serverless.yml

```diff
@@ -271,3 +271,6 @@ discover.enableUiSettingsValidations: true
 xpack.dataUsage.enabled: true
 # This feature is disabled in Serverless until fully tested within a Serverless environment
 xpack.dataUsage.enableExperimental: ['dataUsageDisabled']
+
+## Search Connectors in stack management
+xpack.searchConnectors.enabled: false
```

Comment:
We could keep the default at this level (enabled: true), and let solutions disable it if they want. WDYT?
NB it is already disabled in `serverless.es.yml`, with the old _configPath_ (which should be updated):

```
xpack.search.enabled: false
```

I wonder if we want this enabled on `'chat'` or not. @pgayvallet ?

Reply:
If it's only for serverless right now (or is it not?), shouldn't we set `xpack.searchConnectors.enabled` to `false` directly in the code/config schema (default value) and then only re-enable it in specific serverless configs to ensure it's not unintentionally enabled where we don't expect it?